content/platforms/faqs/_index.md

```diff
@@ -195,7 +167,7 @@ seLinuxContext:
 
 ([latest version on GitHub](https://raw.githubusercontent.com/RedisLabs/redis-enterprise-k8s-docs/master/scc.yaml))
```

Comment:
The "latest version on GitHub" links at lines 168 and 197 lead to a 404, by the way. The removal of the expands looks good to me though.

Reply:
Filed as DOC-829.